src/ansys/fluent/core/utils/fluent_version.py

```diff
@@ -113,8 +115,22 @@ def get_latest_installed(cls):
         AnsysVersionNotFound
             If an Ansys version cannot be found.
         """
+
+        def fluent_exe_exists(awp_var):
+            awp_root = os.getenv(awp_var)
+            if awp_root is None:
+                return False
+
+            fluent_root = Path(awp_root) / "fluent"
+            fluent_exe = (
+                fluent_root / "ntbin" / "win64" / "fluent.exe"
+                if platform.system() == "Windows"
+                else fluent_root / "bin" / "fluent"
+            )
+            return fluent_exe.exists()
+
         for member in cls:
-            if member.awp_var in os.environ:
+            if fluent_exe_exists(member.awp_var):
```

Comment:
@mkundu1 
The following code,
```python
raise AnsysVersionNotFound(
            "Verify the value of the 'AWP_ROOT' environment variable."
        )
```
can become
```python
raise AnsysVersionNotFound()
```
The message, fixed in the exception class, can be something like "Unable to locate a compatible Ansys Fluent installation. Ensure that an environment variable like 'AWP_ROOT242' or 'AWP_ROOT251' points to a supported Ansys version, and that Fluent is included in the installation.".